package.json

```diff
@@ -2,9 +2,18 @@
   "private": true,
   "version": "1.0.0",
   "description": "A template for kick starting a Cloudflare Workers project",
-  "main": "index.js",
+  "main": "index.ts",
```

Comment:
should be `"main": "dist/bundle.js"` .... see the webpack config, that's building and outputting this